src/Sfa.Das.Sas.Indexer.ApplicationServices/Provider/Services/ProviderDataService.cs

```diff
@@ -174,25 +182,51 @@ public async Task<ProviderSourceDto> LoadDatasetsAsync()
 
             await Task.WhenAll(frameworks, standards);
 
-            _logger.Debug($"Finished loading frameworks, standards");
+            _logger.Info($"Provider Indexer, task 4 of {totalProviderTaskCount}: Finished loading frameworks, standards");
 
             var ukprnList = JoinUkprnLists(roatpProviders, activeProviders);
 
+            _logger.Info($"Provider Indexer, task 5 of { totalProviderTaskCount}: {ukprnList?.Count()} ukprns gathered from roatp and active FCS Providers");
+
+            var ukrlpProviders = _mediator.Send(new UkrlpProviderRequest(ukprnList));
```

Comment:
Can these not be `SendAsync` ?

Reply:
In the original code, this call isn't async, and I wanted to do as little with the original code as possible (other than I'd thought I'd take the opportunity to make the rollcall of gathering logged in a better way)